Categories/UIViewController+BAPromise.m

```diff
@@ -0,0 +1,52 @@
+//
+//  UIViewController+BAPromise.m
+//  BAPromise
+//
+//  Created by Ben Allison on 4/21/16.
+//  Copyright © 2016 Ben Allison. All rights reserved.
+//
+
+#import "UIViewController+BAPromise.h"
+#import "BAPromiseClient.h"
+
+@implementation UIViewController (BAPromise)
+
+-(BAPromise *)promise_DismissViewControllerAnimated:(BOOL)animated
```

Comment:
nitpick here but you are using snake_case for first part of ...present... below :-)